src/com/cloudogu/ces/cesbuildlib/K3d.groovy

```diff
@@ -290,8 +290,12 @@ class K3d {
     void configureComponents(components = [:]) {
         def evals = []
         components.each { componentName, componentConfig ->
-            componentConfig.each { configKey, configValue ->
-                evals << ".components.${componentName}.${configKey} = \\\"${configValue}\\\""
+            if (componentConfig == null) {
```

Comment:
In light of what actually should be done here, I feel the condition would read better if the boolean variable would receive a name that reflects the business decision of the code, like so
```suggestion
            def disableComponent=componentConfig == null
            if (disableComponent) {
```